extension/bootstrap.js

```diff
@@ -78,20 +74,23 @@ var PersonasPlusBootstrapAddon = {
         Cu.unload("resource://personas/modules/personas.js");
         this.requestRemovePrerequisites(data);
         if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
-            // modules that come with Firefox
-            Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-            // LightweightThemeManager may not be not available (Firefox < 3.6 or Thunderbird)
-            try {
-                Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
-            } catch (e) {
-                LightweightThemeManager = null;
-            }
-            try {
-                LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
-            } catch (e) {
-                try {
-                    LightweightThemeManager.currentTheme = null;
-                } catch (e) {}
+            if (LightweightThemeManager.currentTheme) {
+                if (LightweightThemeManager.currentTheme.id != "15131") {
+                    var prompt = Components.classes["@mozilla.org/embedcomp/prompt-service;1"].getService(Components.interfaces.nsIPromptService);
+                    if (prompt.confirm(null, "Personas Plus", "Would you like to remove the current theme?")) {
+                        try {
+                            Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
+                            LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
+                        } catch (e) {
+                            try {
+                                LightweightThemeManager.currentTheme = null;
+                            } catch (e) {}
+                        }
+                    }
+                } else {
+                    // Remove Groovy Blue Theme
+                    LightweightThemeManager.forgetUsedTheme("15131");
```

Comment:
you are right, 250 is unnecessary.

When clicking Disable button, theme should be removed. But uninstall function only runs after about:addons page is closed. So it should be handled in Shutdown function.